src/patternfly/components/Tabs/tabs.scss

```diff
@@ -74,17 +92,24 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
   @media screen and (min-width: $pf-global--breakpoint--xl) {
     --pf-c-tabs__scroll-button--Width: var(--pf-c-tabs__scroll-button--xl--Width);
+    --pf-c-tabs--inset: var(--pf-c-tabs--xl--inset);
+    --pf-c-tabs--m-box--inset: var(--pf-c-tabs--m-box--xl--inset);
+    --pf-c-tabs--m-box__item--m-current--first-child__link--before--BorderLeftWidth: var(--pf-c-tabs--m-box__item--m-current--first-child__link--before--xl--BorderLeftWidth);
+    --pf-c-tabs--m-box__item--m-current--last-child__link--before--BorderRightWidth: var(--pf-c-tabs--m-box__item--m-current--last-child__link--before--xl--BorderRightWidth);
```

Comment:
`--pf-c-tabs--m-box__item--m-current--first-child__link--before--xl--BorderLeftWidth` and `--pf-c-tabs--m-box__item--m-current--first-child__link--before--xl--BorderRightWidth` are defined twice. Once under `// Box` and another under `// Link before`

Also looks like `--pf-c-tabs--m-box__item--m-current--first-child__link--before--xl--BorderRightWidth` should be `--pf-c-tabs--m-box__item--m-current--last-child__link--before--xl--BorderRightWidth`